pontoon/sync/core.py

```diff
@@ -228,6 +227,6 @@ def commit_changes(db_project, vcs_project, changeset, locale):
         'authors': set(authors)
     })
 
-    locale_path = locale_directory_path(vcs_project.checkout_path, locale.code)
+    locale_path = vcs_project.locale_directory_paths[locale.code]
```

Comment:
No need to search for locale directory path on each commit - we already have this information.